Modules/Loadable/InteractiveTubesToTree/Logic/vtkSlicerInteractiveTubesToTreeLogic.cxx

```diff
@@ -537,4 +537,72 @@ ::deleteTubeFromSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode, std::set<i
     }  
   }
   spatialNode->UpdatePolyDataFromSpatialObject();
+}
+
+//---------------------------------------------------------------------------=
+bool vtkSlicerInteractiveTubesToTreeLogic
+::GetSpatialObjectRootStatusData(vtkMRMLSpatialObjectsNode* spatialNode, int currTubeID)
+{
+  if (!spatialNode)
+  {
+    return false;
+  }
+  TubeNetType* spatialObject = spatialNode->GetSpatialObject();
+
+  char childName[] = "Tube";
+  TubeNetType::ChildrenListType* tubeList =
+    spatialObject->GetChildren(spatialObject->GetMaximumDepth(), childName);
+
+  for (TubeNetType::ChildrenListType::iterator tubeIt = tubeList->begin(); tubeIt != tubeList->end(); ++tubeIt)
+  {
+    VesselTubeType* currTube =
+      dynamic_cast<VesselTubeType*>((*tubeIt).GetPointer());
+    if (!currTube || currTube->GetNumberOfPoints() < 2)
+    {
+      continue;
+    }
+    if (currTube->GetId() == currTubeID)
+    {
+      return currTube->GetRoot();
+    }
+    else
+    {
+      continue;
```

Comment:
this continue statement isn't needed since no lines follow it in the loop, right?